modules/tinymce/src/themes/silver/main/ts/ui/menus/item/build/CommonMenuItem.ts

```diff
@@ -18,8 +18,7 @@ import { menuItemEventOrder, onMenuItemExecute } from '../ItemEvents';
 import ItemResponse from '../ItemResponse';
 import { ItemStructure } from '../structure/ItemStructure';
 
-export const componentRenderPipeline = (xs: Array<Optional<AlloySpec>>) =>
-  Arr.bind(xs, (o) => o.toArray());
+export const componentRenderPipeline = (xs: Array<Optional<AlloySpec>>) => Optionals.cat(xs);
```

Comment:
```suggestion
export const componentRenderPipeline: (xs: Array<Optional<AlloySpec>>) => Array<AlloySpec> = Optionals.cat;
```
Can we eta reduce this, or just inline it?

Reply:
Yeah, we probably should just remove this and inline it in the 2 places it's used.

Reply:
Worth noting this is one place where ETA reducing is not kind, as you lose the types unless you do what you've suggested above.

Reply:
I used the suggestion here, as this is exported and used in a few cases so inlining it wasn't as clear.